.github/workflows/build-linux-amd64.yml

```diff
@@ -37,31 +37,29 @@ jobs:
           export GOARCH=${GOARCH}
           export GOOS=${GOOS}
           make build-linux-with-shared-library
-          pushd build
-          mkdir -p initia_${VERSION}
-          mv libmovevm.so initia_${VERSION}/libmovevm.${ARCH_NAME}.so
-          mv libcompiler.so initia_${VERSION}/libcompiler.${ARCH_NAME}.so
-          mv initiad initia_${VERSION}/
-          tar -czvf initia_${VERSION}_Linux_${ARCH_NAME}.tar.gz initia_${VERSION}
-          popd
+          mkdir -p release-${GOOS}-${ARCH_NAME}
```

Comment:
I think we need to remove this release-${GOOS}-${ARCH_NAME}. When we unzip it, there should be initiad or bin/initiad directly

Reply:
i;m getting this weird error when I try to statesync 
```error="invalid binaries: error downloading binary for os/arch linux/arm64: url \"xxxx //http://github.com/initia-labs/initia/releases/download/v0.6.0/initia_v0.6.0_Linux_aarch64.tar.gz?checksum=sha256:c5919b1b38717f327fea8e0d376dfd1f4c52fab1423d526692af5637e4f8c6d6%5C" result does not contain a bin/initiad or initiad file" module=cosmovisor```

Reply:
tarball won't have subdirectory starts with release- since I've put `-C release-...` option for tar command